docs-source/content/samples/simple/azure-kubernetes-service/model-in-image.md

```diff
@@ -0,0 +1,836 @@
+---
+title: "Model in Image"
+date: 2020-11-24T18:22:31-05:00
+weight: 3
+description: "Sample for creating a WebLogic cluster on the Azure Kubernetes Service with model in image approach."
+---
+
+This sample demonstrates how to use the [Oracle WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter "the operator") to set up a WebLogic Server (WLS) cluster on the Azure Kubernetes Service (AKS) using the model in image approach. After going through the steps, your WLS domain runs on an AKS cluster instance and you can manage your WLS domain by interacting with the operator.
+
+#### Contents
+
+ - [Prerequisites](#prerequisites)
+ - [Create an AKS cluster](#create-the-aks-cluster)
+ - [Install WebLogic Server Kubernetes Operator](#install-weblogic-server-kubernetes-operator)
+ - [Create Docker image](#create-docker-image)
+ - [Create WebLogic domain](#create-weblogic-domain)
+ - [Invoke the web application](#invoke-the-web-application)
+ - [Rolling updates](#rolling-updates)
+ - [Clean up resource](#clean-up-resources)
+ - [Troubleshooting](#troubleshooting)
+ - [Useful links](#useful-links)
+ 
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/prerequisites-02.txt" >}}
+
+
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/create-aks-cluster-body-01.txt" >}}
+
+##### Clone WebLogic Server Kubernetes Operator repository
+
+Clone the [Oracle WebLogic Server Kubernetes Operator repository](https://github.com/oracle/weblogic-kubernetes-operator) to your machine. We will use several scripts in this repository to create a WebLogic domain. This sample was tested with v3.1.1.
+
+```bash
+$ git clone https://github.com/oracle/weblogic-kubernetes-operator.git
+cd weblogic-kubernetes-operator
+#TODO: we have to fix the branch after the source code are merged
+#git checkout v3.1.1
+```
+
+{{< readfile file="/samples/simple/azure-kubernetes-service/includes/create-aks-cluster-body-02.txt" >}}
+
+> **Note**: If you run into VM size failure, see [Troubleshooting - Virtual Machine size is not supported]({{< relref "/samples/simple/azure-kubernetes-service/troubleshooting#virtual-machine-size-is-not-supported" >}}).
+
+
+#### Install WebLogic Server Kubernetes Operator
+
+The Oracle WebLogic Server Kubernetes Operator is an adapter to integrate WebLogic Server and Kubernetes, allowing Kubernetes to serve as a container infrastructure hosting WLS instances.  The operator runs as a Kubernetes Pod and stands ready to perform actions related to running WLS on Kubernetes.
+
+You must have the `cluster-admin` role to install the operator. The operator does not need the `cluster-admin` role at runtime. Grant the Helm service account the `cluster-admin role` using this configuration file `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/model-in-image/helm-sa-cluster-admin-role.yaml`.
```

Comment:
This is obsoleted. Helm needed it's own service account only for Helm 2.x. With Helm 3.x the service account of the customer running the `helm install` is used.